packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx

```diff
@@ -96,34 +108,26 @@ export const AccountsMenu = () => {
         }
     };
 
+    const startDiscovery = () => {
+        dispatch(startDiscoveryThunk());
+    };
+
     if (!device || !discovery) {
         return (
             <Wrapper>
-                <Scroll>
-                    <MenuHeader>
-                        <AccountSearchBox />
-                    </MenuHeader>
-                    <AccountItemSkeleton />
-                </Scroll>
+                <AccountsMenuNotice>
+                    <Translation id="TR_ACCOUNT_NO_ACCOUNTS" />
+                </AccountsMenuNotice>
             </Wrapper>
         );
     }
 
-    const failed = getFailedAccounts(discovery);
-
-    const list = sortByCoin(accounts.filter(a => a.deviceState === device.state).concat(failed));
-
-    const isEmpty = list.length === 0;
     return (
         <Wrapper>
             <MenuHeader>
                 <Row>
-                    {!isEmpty && <AccountSearchBox />}
-                    <AddAccountButton
-                        data-test="@account-menu/add-account"
-                        device={device}
-                        isFullWidth={isEmpty}
```

Comment:
I believe it should be like this, it's also in the video that Ondra had provided

<img width="200" alt="image" src="https://github.com/trezor/trezor-suite/assets/45338719/08539e1e-0978-49b3-9997-a7c90debefac">